src/bundle/Resources/public/js/scripts/admin.focus.mode.js

```diff
@@ -0,0 +1,61 @@
+(function (global, doc) {
+    let activeFieldEdit = null;
+    const ENABLE_FOCUS_MODE_EVENT_NAME = 'ibexa-focus-mode:on';
+    const DISABLE_FOCUS_MODE_EVENT_NAME = 'ibexa-focus-mode:off';
+    const focusModeEnableBtns = doc.querySelectorAll('.ibexa-field-edit__focus-mode-control-btn--enable');
+    const focusModeDisbaleBtns = doc.querySelectorAll('.ibexa-field-edit__focus-mode-control-btn--disable');
+    const changeFocusModeState = (active) => {
+        if (!activeFieldEdit) {
+            return;
+        }
+
+        const dispatchEventName = active ? ENABLE_FOCUS_MODE_EVENT_NAME : DISABLE_FOCUS_MODE_EVENT_NAME;
+        const editorSourceElement = activeFieldEdit.querySelector('.ibexa-data-source__richtext');
+        const editorInstance = editorSourceElement.ckeditorInstance;
+
+        activeFieldEdit.classList.toggle('ibexa-field-edit--focus-mode-active', active);
+        editorInstance.set('focusModeActive', active);
+
+        doc.body.dispatchEvent(
+            new CustomEvent(dispatchEventName, {
+                detail: {
+                    activeFieldEdit,
+                },
+            }),
+        );
+
+        if (!active) {
+            activeFieldEdit = null;
+        }
+    };
+    const watchDisableFocusModeByKeyboard = (event) => {
```

Comment:
Why we always add Keyboard with func name for keydown but never Click when using click? 😄 
Maybe exitFocusMode || exitFocusModeByKeyboard